v2/emailpassword/troubleshooting/cors-issues.mdx

```diff
@@ -0,0 +1,265 @@
+---
+id: cors-issues
+title: CORS Issues
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./emailpassword/troubleshooting/cors-issues.mdx -->
+
+# CORS Issues 
+
+## Introduction
+
+Let's talk about Cross-Origin Resource Sharing (CORS)!
```

Comment:
The intro can use fonts.com as an example, but it's way too long.. should make it much smaller

Reply:
Ok, cutting down the introduction sections